docs/docs/05-migrations/11-v7-to-v8.md

```diff
@@ -153,20 +166,41 @@ app.BasicModuleManager = module.NewBasicManagerFromManager(
+- The function `IsBound` of controller submodule has been renamed to [`hasCapability`](https://github.com/cosmos/ibc-go/blob/v8.0.0-beta.0/modules/apps/27-interchain-accounts/controller/keeper/keeper.go#L112) and made unexported.
+- The function `IsBound` of host submodule has been renamed to [`hasCapability`](https://github.com/cosmos/ibc-go/blob/v8.0.0-beta.0/modules/apps/27-interchain-accounts/host/keeper/keeper.go#L95) and made unexported.
 
 ## Relayers
 
 - Getter functions in `MsgChannelOpenInitResponse`, `MsgChannelOpenTryResponse`, `MsgTransferResponse`, `MsgRegisterInterchainAccountResponse` and `MsgSendTxResponse` have been removed. The fields can be accessed directly.
 - `channeltypes.EventTypeTimeoutPacketOnClose` (where `channeltypes` is an import alias for `"github.com/cosmos/ibc-go/v8/modules/core/04-channel"`) has been removed, since core IBC does not emit any event with this key.
-- Attribute with key `counterparty_connection_id` has been removed from event with key `connectiontypes.EventTypeConnectionOpenInit` (where `connectiontypes` is an import alias for `"github.com/cosmos/ibc-go/v8/modules/core/03-connection/types"`) and attribute with key `counterparty_channel_id` has been removed from event with key `channeltypes.EventTypeChannelOpenInit` (where `channeltypes` is an import alias for `"github.com/cosmos/ibc-go/v8/modules/core/04-channel"`) since both (counterparty connection ID and counterparty channel ID) are empty on `ConnectionOpenInit` and `ChannelOpenInit` respectively. 
+- Attribute with key `counterparty_connection_id` has been removed from event with key `connectiontypes.EventTypeConnectionOpenInit` (where `connectiontypes` is an import alias for `"github.com/cosmos/ibc-go/v8/modules/core/03-connection/types"`) and attribute with key `counterparty_channel_id` has been removed from event with key `channeltypes.EventTypeChannelOpenInit` (where `channeltypes` is an import alias for `"github.com/cosmos/ibc-go/v8/modules/core/04-channel"`) since both (counterparty connection ID and counterparty channel ID) are empty on `ConnectionOpenInit` and `ChannelOpenInit` respectively.
+- As part of the migration to [governance V1 messages](#governance-v1-migration) the following changes in events have been made:
+
+```diff
+// IBC client events vars
+var (
+  EventTypeCreateClient          = "create_client"
+  EventTypeUpdateClient          = "update_client"
+  EventTypeUpgradeClient         = "upgrade_client"
+  EventTypeSubmitMisbehaviour    = "client_misbehaviour"
+- EventTypeUpdateClientProposal  = "update_client_proposal"
+- EventTypeUpgradeClientProposal = "upgrade_client_proposal"
++ EventTypeRecoverClient              = "recover_client"
++ EventTypeScheduleIBCSoftwareUpgrade = "schedule_ibc_software_upgrade"
+  EventTypeUpgradeChain               = "upgrade_chain"
+)
+```
```

Comment:
This part is subject to change due to https://github.com/cosmos/ibc-go/pull/4729.